public/pages/workflows/new_workflow/quick_configure_optional_fields.tsx

```diff
@@ -211,6 +212,7 @@ export function QuickConfigureOptionalFields(
             {unknownEmbeddingLength && (
               <>
                 <EuiSpacer size="s" />
+                {props.workflowType !== WORKFLOW_TYPE.NEURAL_SPARSE_SEARCH && (
```

Comment:
how about add this conditional on line 212 so we are consistent with the spacing and not unnecessarily rendering an `EuiSpacer` component`?

```
{unknownEmbeddingLength && props.workflowType !== WORKFLOW_TYPE.NEURAL_SPARSE_SEARCH && (
```